src/app.ts

```diff
@@ -56,6 +56,8 @@ import {
     handleLike
 } from './dispatchers';
 
+import { searchAction } from './api/action/search';
```

Comment:
@allouis I took a slightly different approach on this action and placed it here. It feels like the handlers file is starting to get a bit unwieldily. Thoughts?